db/migrate/20150902194325_create_items.rb

```diff
@@ -0,0 +1,11 @@
+class CreateItems < ActiveRecord::Migration
+  def change
+    create_table :items do |t|
+      t.references :meal
+      t.string :name
+      t.float :price
```

Comment:
We should be storing `price` as a positive integer denoting cents. Floats always get messy.